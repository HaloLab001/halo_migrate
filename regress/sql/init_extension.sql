SET client_min_messages = warning;
CREATE EXTENSION halo_migrate;
RESET client_min_messages;
