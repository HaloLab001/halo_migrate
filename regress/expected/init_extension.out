SET client_min_messages = warning;
CREATE EXTENSION pg_migrate;
RESET client_min_messages;
